doc/examples/auto_csfle_example.py

```diff
@@ -0,0 +1,105 @@
+import os
```

Comment:
Have you tested the two examples here? It would be good to confirm that they work as expected.

Reply:
yep I ran both of these scripts